src/libs/OptionsListUtils.js

```diff
@@ -531,6 +544,13 @@ function createOption(accountIDs, personalDetails, report, reportActions = {}, {
             });
         }
 
+        const lastAction = visibleReportActionItems[report.reportID];
+        const shouldDisplayLastActorName = lastAction && lastAction.actionName !== CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW && lastAction.actionName !== CONST.REPORT.ACTIONS.TYPE.IOU;
+
+        if (shouldDisplayLastActorName && lastActorDisplayName && lastMessageTextFromReport) {
+            lastMessageText = `${lastActorDisplayName}: ${lastMessageTextFromReport}`;
```

Comment:
Let's use common function whether to add prefix or not and use it in both here and SidebarUtils

https://github.com/Expensify/App/blob/3ee24ec7868be996c78e1ae8097899ebdf33a709/src/libs/SidebarUtils.ts#L401-L402

We should make `alternateText` consistent between LHN and search page.